blog/01-18-2023-pros-and-cons-of-celo-api/pros-and-cons-of-celo-api.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Pros and cons of using Celo API to create Defi applications
+description: A foray in to building on Celo using Celo's API
+authors:
+  - name: Maxwell Onyeka
+    title: Technical writer 
+    url: https://github.com/maxzysparks
```

Comment:
Remove the URL in this section

Reply:
Hi, I could see this url in other sages submission, is there a reason why mine is not needed?